### ISSUE TITLE
sidebar does not show if started with the -y flag

### ISSUE THREAD

**Issue by rnagy** [neomutt/neomutt#1971]
The sidebar is not being displayed if the -y flag is used.

## Steps to Reproduce

1. make sure sidebar is enabled
2. neomutt -y

## How often does this happen?

- Always

## When did it start to happen?

Upgrade to 20191111

```
NeoMutt 20191111
Copyright (C) 1996-2016 Michael R. Elkins and others.
NeoMutt comes with ABSOLUTELY NO WARRANTY; for details type 'neomutt -vv'.
NeoMutt is free software, and you are welcome to redistribute it
under certain conditions; type 'neomutt -vv' for details.

System: OpenBSD 6.6 (amd64)
ncurses: ncurses 5.7.20081102 (compiled with 5.7.20081102)
libiconv: 1.16
libidn2: 2.0.0 (compiled with 2.0.0)
hcache backends: kyotocabinet

Compiler:
OpenBSD clang version 8.0.1 (tags/RELEASE_801/final) (based on LLVM 8.0.1)
Target: amd64-unknown-openbsd6.6
Thread model: posix
InstalledDir: /usr/bin

Configure options: --mandir=/usr/local/man --with-ui=ncurses --docdir=/usr/local/share/doc/neomutt --with-lock=flock --idn=0 --idn2 --fmemopen --ssl --kyotocabinet

Compilation CFLAGS: -O2 -pipe -g  -I/usr/local/include -std=c99 -D_ALL_SOURCE=1 -D_GNU_SOURCE=1 -D__EXTENSIONS__ -DNCURSES_WIDECHAR

Default options:
  +attach_headers_color +compose_to_sender +compress +cond_date +debug 
  +encrypt_to_self +forgotten_attachments +forwref +ifdef +imap +index_color 
  +initials +limit_current_thread +multiple_fcc +nested_if +new_mail +nntp +pop 
  +progress +quasi_delete +regcomp +reply_with_xorig +sensible_browser +sidebar 
  +skip_quoted +smtp +status_color +timeout +tls_sni +trash 

Compile options:
  -autocrypt +bkgdset +color +curs_set -fcntl +flock +fmemopen +futimens 
  +getaddrinfo -gnutls -gpgme -gss +hcache -homespool +idn -inotify 
  -locales_hack -lua +meta -mixmaster +nls -notmuch +openssl +pgp -sasl +smime 
  -sqlite +start_color +sun_attachment +typeahead 
MAILPATH="/var/mail"
PKGDATADIR="/usr/local/share/neomutt"
SENDMAIL="/usr/sbin/sendmail"
SYSCONFDIR="/usr/local/etc"

To learn more about NeoMutt, visit: https://neomutt.org
If you find a bug in NeoMutt, please raise an issue at:
    https://github.com/neomutt/neomutt/issues
or send an email to: <neomutt-devel@neomutt.org>
```

**Comment by jindraj** [comment]
starting neomutt with -y option won't display sidebar but file browser listing of defined mailboxes.
Have you tried `:set sidebar_visible` inside neomutt? It seems you don't have it set in your configuration file.

**Comment by rnagy** [comment]
> starting neomutt with -y option won't display sidebar but file browser listing of defined mailboxes.
> Have you tried `:set sidebar_visible` inside neomutt? It seems you don't have it set in your configuration file.

I've tried `:set sidebar_visible` and the sidebar does not show up.

Reverting to a previous (neomutt-20180716) makes it work just fine
with the same configuration file.

**Comment by flatcap** [comment]
**tl;dr** The sidebar isn't functional in the browser.  It shouldn't be there.

The sidebar is a late addition to Mutt.  The code was maintained, for many years, without the support of Mutt, which meant the authors had to compromise on its integration.  It's now part of Mutt, but it's still poorly integrated.

If you'd like to the worst example, hit <kbd>?</kbd> for the help.  The sidebar really shouldn't be there!

In the near future, the sidebar will disappear from more places, only being visible when it's _usable_.
